Source/Filters/Photo/YPCoverImageView.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  YPCoverImageView.swift
+//
+//
+//  Created by Will Saults on 1/15/24.
+//
+
+import UIKit
+import Photos
+
+public class YPCoverImageView: YPAdjustableView {
+
+    internal var coverImageView = UIImageView()
```

Comment:
Does this need to be explicitly internal?

Reply:
Nope, that can come out. Thanks!